drivers/spi/spi_ll_stm32.c

```diff
@@ -725,8 +715,8 @@ static int32_t spi_stm32_count_bufset_frames(const struct spi_config *config,
 }
 
 static int32_t spi_stm32_count_total_frames(const struct spi_config *config,
-	                                    const struct spi_buf_set *tx_bufs,
-	                                    const struct spi_buf_set *rx_bufs)
+	const struct spi_buf_set *tx_bufs,
+	const struct spi_buf_set *rx_bufs)
```

Comment:
Identation was fine before

Reply:
Same as above, indentation check fails as explained in https://github.com/zephyrproject-rtos/zephyr/pull/66503#discussion_r1426785103